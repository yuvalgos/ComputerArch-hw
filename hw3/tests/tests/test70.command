./dflow_calc tests/test70.dat tests/test70.in p88 d17 p195 d112 d39 d193 d109 p15 p26 p40 d197 d143 d162 d96 d91 p14 d25 p126 p125 d193 p23 d90 d6 p3 d182 d71 d168 p35 p113 p80 d58 p4 d23 d114 p47 p185 d177 p80 p27 p24 d124 d74 p133 p50 p31 d53