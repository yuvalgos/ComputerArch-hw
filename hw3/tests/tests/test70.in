6 11 16 28
6 24 9 1
0 26 16 11
0 19 24 6
3 7 0 25
2 11 3 2
6 11 31 3
2 15 28 24
0 8 4 27
1 4 26 5
1 14 23 22
3 17 2 2
8 10 21 25
2 24 1 18
2 19 8 3
0 10 2 26
6 29 21 0
0 11 16 6
0 26 25 6
4 20 27 31
7 15 13 12
0 27 7 23
5 29 10 3
8 23 1 16
3 22 10 9
1 11 9 10
5 21 27 26
4 1 18 11
0 21 2 20
3 31 8 17
4 15 29 31
1 11 4 30
3 15 4 12
7 3 13 16
7 26 4 25
3 21 26 20
1 5 6 24
6 10 3 19
4 9 22 3
1 13 20 23
7 3 0 9
3 25 31 3
0 28 13 25
7 8 27 28
7 23 7 31
3 23 29 2
3 9 15 14
1 26 25 2
2 20 10 17
6 21 14 0
5 30 28 24
0 11 8 19
3 0 7 24
4 31 27 17
7 12 12 14
3 9 17 0
1 8 11 31
6 1 1 20
0 26 31 11
7 17 23 2
1 25 7 8
2 4 11 28
3 10 6 6
0 20 24 30
6 3 0 2
1 14 17 21
4 6 23 5
6 20 10 9
3 21 6 27
4 11 6 3
2 21 20 24
7 2 20 21
1 8 19 12
7 31 24 16
2 28 17 16
0 24 21 4
6 31 7 27
7 24 13 6
8 26 30 11
1 7 12 10
2 23 23 24
7 21 31 31
4 31 9 17
1 18 18 22
5 4 9 15
2 3 25 26
0 26 31 1
6 17 23 0
2 8 9 20
1 13 31 27
6 7 30 12
5 21 20 23
2 25 20 0
2 19 6 28
0 11 27 30
7 16 18 31
2 21 3 10
2 17 10 27
6 7 20 6
5 13 9 10
8 24 21 30
0 1 6 24
5 29 3 17
8 3 17 13
8 6 21 26
7 20 28 21
2 12 7 3
3 27 6 27
7 29 21 26
8 2 15 14
1 29 29 5
0 26 23 28
2 10 15 27
4 20 10 28
2 26 29 16
7 19 5 12
0 0 24 29
6 17 30 5
8 10 7 3
5 30 29 22
6 24 1 12
6 12 28 6
0 21 22 9
5 23 27 20
7 2 21 10
8 7 5 28
0 3 23 16
3 17 18 21
7 0 21 18
6 2 28 19
5 16 15 5
4 24 20 9
7 3 28 9
0 19 15 4
2 22 22 7
8 29 13 11
8 27 21 7
5 6 2 7
1 29 4 4
2 20 5 2
1 28 1 28
3 20 4 7
4 25 9 18
6 24 31 16
4 27 29 31
6 6 6 25
0 16 29 7
6 27 17 7
7 10 25 13
0 28 26 20
6 6 22 25
8 16 17 30
8 14 13 31
0 24 29 30
6 8 16 8
3 1 24 17
3 22 20 8
8 13 27 22
2 12 19 4
7 24 18 28
2 28 30 18
6 6 22 21
6 22 8 18
6 19 8 23
4 1 10 3
0 3 24 31
0 8 0 24
6 9 2 14
2 16 25 20
6 29 0 2
6 15 13 3
8 25 10 5
2 27 16 24
6 29 4 30
5 0 25 27
7 11 31 15
2 24 21 7
1 28 30 15
0 17 11 6
3 6 27 14
3 18 19 17
3 5 17 13
1 16 24 29
8 24 1 15
1 29 18 15
4 21 18 29
5 30 27 16
6 6 11 27
5 9 29 14
5 22 8 25
3 22 18 16
4 12 7 16
6 21 9 26
0 6 6 25
7 20 31 23
6 16 20 3
2 9 12 9
4 23 15 14
7 27 13 6
4 24 11 0
