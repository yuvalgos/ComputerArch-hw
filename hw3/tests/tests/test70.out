Reading the opcodes latency info from tests/test70.dat ... Got latency for 9 opcodes
Reading the program file tests/test70.in ... Found 200 instructions
getProgDepth()==296
getDepDepth(88)==38
getInstDeps(17)=={-1,-1}
getDepDepth(195)==191
getInstDeps(112)=={32,107}
getInstDeps(39)=={19,23}
getInstDeps(193)=={187,171}
getInstDeps(109)=={32,65}
getDepDepth(15)==19
getDepDepth(26)==9
getDepDepth(40)==39
getInstDeps(197)=={170,152}
getInstDeps(143)=={82,130}
getInstDeps(162)=={154,83}
getInstDeps(96)=={85,62}
getInstDeps(91)=={67,80}
getDepDepth(14)==2
getInstDeps(25)=={-1,15}
getDepDepth(126)==172
getDepDepth(125)==65
getInstDeps(193)=={187,171}
getDepDepth(23)==0
getInstDeps(90)=={50,54}
getInstDeps(6)=={-1,-1}
getDepDepth(3)==17
getInstDeps(182)=={176,173}
getInstDeps(71)=={67,70}
getInstDeps(168)=={142,141}
getDepDepth(35)==41
getDepDepth(113)==148
getDepDepth(80)==71
getInstDeps(58)=={53,51}
getDepDepth(4)==0
getInstDeps(23)=={-1,-1}
getInstDeps(114)=={110,95}
getDepDepth(47)==65
getDepDepth(185)==266
getInstDeps(177)=={119,170}
getDepDepth(80)==71
getDepDepth(27)==24
getDepDepth(24)==21
getInstDeps(124)=={122,118}
getInstDeps(74)=={59,-1}
getDepDepth(133)==42
getDepDepth(50)==67
getDepDepth(31)==22
getInstDeps(53)=={21,11}
